src/pages/login/index.tsx

```diff
@@ -21,7 +21,7 @@ const Login: NextPage = () => {
       {isMember ? (
         <LoginForm newMember={handleMember} />
       ) : (
-        <SignUp userClickedBack={handleBack} />
+        <SignUp goBack={handleBack} />
```

Comment:
We usually use the `on` prefix for callbacks in React properties. There are exceptions to that convention, but in this case I see no reason not to call this property `onClickBack` or something like that.